tests/unit/test-cmd/test.run.js

```diff
@@ -127,6 +127,19 @@ describe('run', () => {
     });
   });
 
+  it('passes url to Firefox when specified', () => {
+    const cmd = prepareRun();
+    const {firefoxApp} = cmd.options;
+
+    return cmd.run({startUrl: 'www.example.com'}).then(() => {
+      assert.equal(firefoxApp.run.called, true);
+      assert.equal(firefoxApp.run.firstCall.args[1].binaryArgs[0],
```

Comment:
we can probably use [`assert.deepEqual`](http://chaijs.com/api/assert/#method_deepequal) to test `binaryArgs` with a nicer single assertion.